doc/dune-files.rst

```diff
@@ -1042,28 +1030,29 @@ See the :ref:`user-actions` section for more details.
   ``(targets)`` can be omitted if it can be inferred from the action. See
   `inferred rules`_.
 
-- ``(deps <deps-conf list>)``, to specify the dependencies of the
-  rule. See the :ref:`deps-field` section for more details.
+- ``(deps <deps-conf list>)``, to specify the dependencies of the rule. See the
```

Comment:
You have “to specify” here and the following one, then “specifies” after that. I’d go with “specifies” on all without the comma after the list item, like:

…``(deps <deps-conf list>)`` specifies the …